src/core/map.js

```diff
@@ -1194,11 +1226,14 @@ geo.map = function (arg) {
    * Cancel any existing transition.  The transition will send a cancel event
    * at the next animation frame, but no further activity occurs.
    *
+   * @param {string} source option cause of the cancel.
```

Comment:
Could you describe this argument a bit more.  It seems you are expecting a string like `<method name>.<action>`.  Can this method be called without a `source` argument?